app/segments/drill/[id]/submission/input.js

```diff
@@ -655,21 +650,7 @@ export default function Input({ drillInfo, setToggleResult, setOutputData }) {
                 </View>
 
                 {/*Navigation Bottom Sheet */}
-                <BottomSheetModal
-                  ref={navModalRef}
-                  enableDynamicSizing
-                  backdropComponent={({ animatedIndex, style }) => {
-                    return (
-                      <BottomSheetBackdrop
-                        appearsOnIndex={0}
-                        disappearsOnIndex={-1}
-                        animatedIndex={animatedIndex}
-                        style={[style, { top: -insets.top }]}
-                      />
-                    );
-                  }}
-                  backgroundStyle={{ backgroundColor: themeColors.background }}
-                >
+                <BottomSheetWrapper ref={navModalRef}>
                   <BottomSheetScrollView>
```

Comment:
This is probably my mistake as I had a scroll view initially when we had more content for the description but now it will get cut off weird if you scroll. I can resolve this if you want. Up to you. 
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/48861b1b-1d31-48c9-921a-f0ce208b60fa)

Reply:
Ye can you remove them then?

Reply:
I have changed it to be just a bottom sheet view. It looks like with the dynamic sizing it fills the screen now for the description.
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/9078ae21-9049-41a1-bc15-e42e4c60c04d)

Not sure if this is the behavior we want.

Reply:
That covers too much of the screen in my opinion. I also don't think it needs to be dynamically sized. Can just snap multiple snap points